setup.cfg

```diff
@@ -1,5 +1,5 @@
 [pytest]
-addopts = --reuse-db --instafail --showlocals
+addopts =-vvs --reuse-db --instafail --showlocals --tb=short
```

Comment:
ugh, that's what instafail is for! I'll try and see how the results look like without.